Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Beckn/ACL/OnSelect.hs

```diff
@@ -106,7 +106,7 @@ mkQuoteEntities dReq quote = do
 mkFulfillment :: DOnSelectReq -> DQuote.DriverQuote -> OS.FulfillmentInfo
 mkFulfillment dReq quote = do
   let fromLocation = dReq.searchRequest.fromLocation
-  let toLocation = dReq.searchRequest.toLocation
+  let toLocation = last dReq.searchRequest.toLocation
```

Comment:
Why it is a list now?

Reply:
later on we'll be adding stops so that will be a part of this list